src/components/signup-view/signup-view.jsx

```diff
@@ -0,0 +1,76 @@
+import { useState } from "react";
+
+export const SignupView = () => {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const [email, setEmail] = useState("");
+  const [birthday, setBirthday] = useState("");
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const data = {
+      Username: username,
+      Password: password,
+      Email: email,
+      Birthday: birthday,
+    };
+
+    fetch("https://movie-geeks-one.vercel.app/users", {
+      method: "POST",
+      body: JSON.stringify(data),
+      headers: {
+        "Content-Type": "application/json",
+      },
+    }).then((response) => {
+      if (response.ok) {
+        alert("Signup successful");
+        window.location.reload();
+      } else {
+        alert("Signup failed");
+      }
+    });
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <label>
+        Username:
+        <input
+          type="text"
+          value={username}
+          onChange={(e) => setUsername(e.target.value)}
+          required
+          minLength="5"
```

Comment:
Again this one is not enforced in the API and it’s also bad practice to enforce a min length on names in general. This should probably be removed so your API can handle any incorrect input.